qiskit_experiments/library/characterization/analysis/drag_analysis.py

```diff
@@ -182,11 +189,11 @@ def _evaluate_quality(self, fit_data: curve.FitData) -> Union[str, None]:
             - an error on the drag beta smaller than the beta.
         """
         fit_beta = fit_data.fitval("beta")
-        fit_freq0 = fit_data.fitval("freq0")
+        fit_freq = fit_data.fitval("freq")
 
         criteria = [
             fit_data.reduced_chisq < 3,
-            fit_beta.nominal_value < 1 / fit_freq0.nominal_value,
+            fit_beta.nominal_value < 1 / fit_freq.nominal_value,
```

Comment:
The current criteria allow an arbitrarily large negative beta. Perhaps this:

```suggestion
            abs(fit_beta.nominal_value) < 1 / fit_freq.nominal_value / 2,
```